rest-api-spec/build.gradle

```diff
@@ -28,100 +28,102 @@ testClusters.all {
 
 tasks.named("test").configure { enabled = false }
 tasks.named("jarHell").configure { enabled = false }
-
+def willNotFixTests = {
+  return ['cat.aliases/10_basic/Alias against closed index',
```

Comment:
I think we can use wildcards in black lists. I would suggest to wildcard as much of these cat tests as possible to help with brevity and future cases. Also a brief mention that compatibility is not supported for cat APIs would help